bigbluebutton-html5/imports/ui/components/audio/local-echo/service.js

```diff
@@ -27,8 +27,14 @@ const playEchoStream = async (stream, loopbackAgent = null) => {
         loopbackAgent.stop();
       }
     }
-
-    audioElement.srcObject = streamToPlay;
+    const audioCtx = new AudioContext();
+    const source = audioCtx.createMediaStreamSource(stream);
+    const delayNode = new DelayNode(audioCtx, {delayTime: 0.5, maxDelayTime: 2});
```

Comment:
Would be nice if initial delay and max delay buffer were configurable parameters - with 0 as _no delay_.